### ISSUE TITLE
Party invites not shown: 2 JavaScript Errors

### ISSUE THREAD

**Issue by meshachjackson** [HabitRPG/habitica#1597]
![screen shot 2013-10-06 at 2 24 32 pm](https://f.cloud.github.com/assets/624763/1277564/b603e3fe-2ecd-11e3-99b5-1a14e99d5dd6.png)
## Steps to reproduce:
- login
- click 'options' > 'groups'
- enter user code into Members > Invite form field: 
  > eg:  84f8fd20-a060-4332-b487-37a5540439b6
- click invite
## Console Error

> POST https://habitrpg.com/api/v1/groups/c1c980fd-fbb9-452e-9d6e-b20ee8117a7b/invite?uuid=84f8fd20-a060-4332-b487-37a5540439b6 400 (Bad Request) app-571bf273.js:4

Note: Expanding the object reveals that the api returns a status 400 for ""User already pending invitation."

I don't believe this is the same issue, but thought I'd call it out.

The bottom line of the behavior I'm seeing is that the members of the party that I've created are not showing up, even when I get (the first time) confirmation of the invite being sent.

As I've spoken with my team, they also are not seeing the invites.

**Comment by paglias** [comment]
The error you are seeing is because they have already been invited to your
party. They won't show up until they accept the invite.

Are you sure they checked their groups tab for your invite?
Il 06/ott/2013 23:28 "Meshach Jackson" notifications@github.com ha
scritto:

**Comment by paglias** [comment]
Probably the same ad #1589, let's follow up there
